assets/src/apps/authoring/components/PropertyEditor/custom/CustomDropdownTemplate.tsx

```diff
@@ -0,0 +1,59 @@
+import React, { Fragment, useState } from 'react';
+import { SequenceDropdown } from './SequenceDropdown';
+import {
+  findInHierarchy,
+  getHierarchy,
+  SequenceEntry,
+  SequenceEntryChild,
+} from 'apps/delivery/store/features/groups/actions/sequence';
+import { selectSequence } from 'apps/delivery/store/features/groups/selectors/deck';
+import { useSelector } from 'react-redux';
+import { useEffect } from 'react';
+import { UiSchema } from '@rjsf/core';
+
+interface CustomDropdownProps {
+  id: string;
+  label: string;
+  uiSchema: UiSchema;
+  description: string;
+  properties: object;
+  value: string;
+  onChange: (value: string) => void;
+}
+const CustomDropdownTemplate: React.FC<CustomDropdownProps> = (props) => {
+  const { id, label, uiSchema, description, properties, value, onChange } = props;
+  const sequence = useSelector(selectSequence);
+  const hierarchy = getHierarchy(sequence);
+  const [buttonLabel, setButtonLabel] = useState<string>('');
+  const [val, setVal] = useState<string>(value);
+  useEffect(() => {
+    const seq = findInHierarchy(hierarchy, val);
+    setButtonLabel(seq?.custom.sequenceName || 'Next');
+  }, [val]);
+  const onChangeHandler = (item: SequenceEntry<SequenceEntryChild>) => {
+    setVal(item.custom.sequenceId);
+    onChange(item.custom.sequenceId);
```

Comment:
how do you pick `next` in this one? (the data I think needs to be lowercase `next` even if the label is `Next`)